.mergify.yml

```diff
@@ -355,31 +355,6 @@ pull_request_rules:
               - "status-success=ci/centos/upgrade-tests-cephfs"
               - "status-success=ci/centos/upgrade-tests-rbd"
               - "status-success=DCO"
```

Comment:
Ineeed, the **devel + `ready-to-merge`** rule will never be met, as GitHub requires 2 (real) reviews for PRs on the devel branch, Mergify is aware of that requirement automatically.